### ISSUE TITLE
Image Manifest Metadata Lost

### ISSUE THREAD

**Issue by clushie** [nlewo/nix2container#116]
Thank you for nix2container.

If I'm using `pullImage` and check the json it doesn't contain data like  ` Env` or `Labels` or `Cmd` I believe.

Which is sometimes useful information when packaging a docker image.

**Comment by nlewo** [comment]
Yes, it would be nice and this should be pretty to add: https://github.com/nlewo/containers-image-nix/blob/9d7f33ef0058f4df4c0912025f43c758a3289d76/nix/image.go#L150

**Comment by clushie** [comment]
Thanks for pointing me to the right line, I was already reading the `NewImageFromDir` func today, but didn't notice the obvious todo. I'll see if I can make a working PR tomorrow.
